modules/root/root_api.rb

```diff
@@ -1,6 +1,7 @@
 class Proxy::RootApi < Sinatra::Base
   helpers ::Proxy::Helpers
   authorize_with_trusted_hosts
+  authorize_with_ssl_client
```

Comment:
I don't think we need any authorization (including trusted hosts) here. We only available data is proxy features and version.